### ISSUE TITLE
Docs website itself very slow and frequently crashes the tab.

### ISSUE THREAD

**Issue by OpenCoderX** [binance-us/binance-us-api-docs#131]
https://docs.binance.us

When I try to browse the docs website it almost always crashes, scrolls slowly, etc.

Does anyone else experience this?

**Comment by ponsaravanan2021** [comment]
Yes it does. I wanted a way to access it offline. not sure there is a way. tried printing as a pdf but crashes as well
